### PR TITLE
docs: use shared content

### PR DESCRIPTION
**Pull request by bmorelli25**
**Do not merge this PR yet. Things are likely to 💥**

## Summary

This PR removes shared ECS documentation and replaces it with the shared content that lives in `elastic/ecs-logging`.
I'm not sure what needs to stay in step 3, so I commented things out.

## Testing

1. Checkout https://github.com/elastic/ecs-logging/pull/40:
```
cd ../ecs-logging
gh pr checkout 40
```

2. Checkout this PR:
```
cd ../ecs-logging-java
gh pr checkout 117
```

3. In the ecs-logging-java directory, run:
```
$GIT_HOME/docs/build_docs --doc $GIT_HOME/ecs-logging-java/docs/index.asciidoc --resource=$GIT_HOME/ecs-logging/docs/ --chunk 1 --open
```

### CODE REVIEW

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

File: docs/intro.asciidoc [hunk 1]
```diff
@@ -36,87 +25,7 @@ They make it easy to format your logs into ECS-compatible JSON. For example:
...
-
-*Efficient Elasticsearch mappings*::
-+
---
-Leverage Filebeat's default ECS-compatible {filebeat-ref}/configuration-template.html[index template]
---
+Ready to jump in? See the <<setup,get started>> guide.
 
 [float]
 === Mapping
```
Comment (bmorelli25) [inline docs/intro.asciidoc:31]: Can this section move to the `Structured logging with log4j2` page?

**Comment by bmorelli25** [comment]
Per zoom:

* Small intro paragraph with an example
* Import Filebeat instructions from ecs-logging

To do in a follow up PR (same day as merge)
* Link to ECS logging page

Questions:

* See https://github.com/elastic/ecs-logging-java/pull/117#discussion_r539610723 about mapping section.

File: docs/intro.asciidoc (hunk 1 above)
Comment (felixbarny) [reply docs/intro.asciidoc]: I've made it generic and added it to the overview docs. See https://github.com/elastic/ecs-logging/pull/40/commits/c1d11d1c126f27aca375307c0f6d0b7d8b2b2bd3.

**Review by felixbarny — APPROVED** [review]

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

File: docs/intro.asciidoc [hunk 2]
```diff
@@ -1,169 +1,10 @@
 [[intro]]
 == Introduction
 
-Centralized logging for Java applications with the Elastic stack made easy.
+ECS loggers are formatter/encoder plugins for your favorite logging libraries.
+They make it easy to format your logs into ECS-compatible JSON.
 
+TIP: Want to learn more about ECS, ECS logging, and other available language plugins?
+See LINK_TO_ECS_LOGGING_DOCS.
```
Comment (bmorelli25) [inline docs/intro.asciidoc:8]: I'll update this in an immediate follow-up PR. I can't link to this new book until the corresponding docs PR has been merged

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

File: docs/intro.asciidoc (hunk 2 above)
Comment (bmorelli25) [reply docs/intro.asciidoc]: Nevermind, was able to change merge order and get this fix into this PR

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

**Comment by bmorelli25** [comment]
Let's try this one more time. I think I fixed the problem in elastic/docs#2021.

@elasticmachine, run elasticsearch-ci/docs

**Pull request merged**